tests/garage/tf/algos/test_vpg.py

```diff
@@ -9,6 +10,7 @@
 class TestVPG(TfGraphTestCase):
     def test_vpg_cartpole(self):
         """Test VPG with CartPole-v1 environment."""
+        logger.reset_output(TensorBoardOutput())
```

Comment:
there's no point in logging to TensorBoard in the CI, so I think you can just remove all of these (or replace them with a NullOutput)